src/app/modules/consultations/consultation-profile/consultation-questionnaire/consultation-questionnaire.component.ts

```diff
@@ -88,6 +88,11 @@ export class ConsultationQuestionnaireComponent
   responseStatus = 0;
   profaneWords = [];
   environment: any = environment;
+  tooltipTranslations = {
```

Comment:
Why are these not in translation file

Reply:
@Bandinikhil - why are you unable to do so? Can you share any errors or roadblaocks you are facing?